rotkehlchen/db/history_events.py

```diff
@@ -500,10 +502,17 @@ def get_history_events(
                     deserialized_event = EthDepositEvent.deserialize_from_db(data)
 
                 else:
-                    data = entry[data_start_idx:HISTORY_BASE_ENTRY_LENGTH + 1]
-                    deserialized_event = HistoryEvent.deserialize_from_db(entry[data_start_idx:])
+                    data = entry[data_start_idx:]
+                    deserialized_event = HistoryEvent.deserialize_from_db(data)
             except (DeserializationError, UnknownAsset) as e:
-                log.debug(f'Failed to deserialize history event {entry} due to {e!s}')
+                log.error(f'Failed to deserialize history event {entry} due to {e!s}')
+                if msg_aggregator is not None:
+                    msg_aggregator.add_error(
+                        f'Could not deserialize history event: {data[0]}. ' + (
```

Comment:
Okay this is also too much. Unless I misunderstand what data[0] is.

Keep in mind. `msg_aggregator.add_error()` pops up as an error popup to the user. That means it need to be short and succinct.

What we need to show to them is that their DB is probably messed up through an error of theirs and they need to take action. And we should have more details in a log error.